package.json

```diff
@@ -7,11 +7,14 @@
     "node": ">=12"
   },
   "scripts": {
-    "test": "mocha test"
+    "lint": "eslint --report-unused-disable-directives --ignore-path .gitignore .",
+    "fix": "npm run eslint -- --fix",
+    "mocha": "mocha",
```

Comment:
Is this needed for anything?

Reply:
Yeah, so that one can do `npm run mocha`.

Reply:
Ah ok, I didn't know that was a thing people did :)